characters.py

```diff
@@ -128,6 +143,44 @@ def heal(self, other):
         """
         min(other.current_hp + self.healing_power, other.max_hp)
 
+
+class Voodoo(Character, MagicType):
+    def __init__(self, poison_nova, name):
+        MagicType.__init__(self)
+        Character.__init__(self, 1000, name)
+        self.poison_nova = poison_nova
+
+    def act(self, other):
+        self.poison(other)
+        self.remove_if_dead(other)
+
+    def poison(self, other):
+        for modifier in other.modifier_list:
```

Comment:
Nie poprawiłeś tego.
Wkładasz tyle poisonów ile atakowany ma już przypisanych modifierow innego rodzaju.

Reply:
a teraz?

Reply:
Nadal źle. Poprawiasz nie to co trzeba.
Masz sprawdzić czy wśród modifierów jest Poisson i jeśli jest to go zmienić, a jeśli nie ma to dodać nowy.

Ty robisz coś takiego ale wiele razy dla jednego bohatera. A ma być raz. Nie wiem po co Ci w ogóle pętla "for modifier in other.modifier_list".